packages/sign-client/package.json

```diff
@@ -27,7 +27,8 @@
     "test:concurrency": "vitest run --dir test/concurrency",
     "test:xregion": "vitest run --dir test/xregion",
     "test": "npm run test:pre; npm run test:run",
-    "test:ignoreUnhandled": "npm run test:pre; npm run test:run -- --dangerouslyIgnoreUnhandledErrors --no-threads",
+    "test:integration": "vitest run --dir test/sdk/integration",
+    "test:ignoreUnhandled": "npm run test:pre; npm run test:integration -- --dangerouslyIgnoreUnhandledErrors --no-threads",
```

Comment:
Is this still being used btw? This was added as a workaround a while back and the unhandled error issue was resolved since then I think.

Reply:
the `type bool of undefined` or something coming from some inner library still shows up occasionally